src/webapp/app.py

```diff
@@ -241,16 +244,18 @@ def graph():
         # user's session. This shouldn't be cached across user sessions.
         http_cache: dict = session.get("msal_http_response_cache", {})
 
-        # Create an MSAL client (using app configuration values) and providing
-        # our token cache.
+        #<ms_docref_configure_app>
+        # Create an MSAL client using the app's configuration values and provide the token cache.
```

Comment:
This comment is now too long and violates the standard formatting that gets applied when running `black` (and validated with `pylint`).  Can you reapply formatting for consistency?  Thanks.

Reply:
@ckittel sure, I've resolved this in https://github.com/Azure-Samples/ms-identity-docs-code-python/pull/9/commits/1139dd0b17773ce6d4cb0619245e6d5c03166450